src/settings.c

```diff
@@ -349,26 +378,18 @@ void load_settings(const char *cmdline_config_path)
                 "Transparency. range 0-100"
         );
 
-        {
-                char *c = g_strdup(option_get_string(
+        settings.sep_color = parse_sepcolor(option_get_string(
```

Comment:
I don't think having duplicate `option_get_*` calls is a good idea, for once it duplicates the descriptions and general calls but also it should theoretically result in a duplicated entry in the help menu (couldn't make that bug appear in a quick test but I'm not sure why).

Reply:
Ah, thanks for this. That's a good point.

Reply:
Err, see here: 32431bf1cba938710f05d646eb515f3fdbb7f19b (`Move separator color into single struct`), there it got removed and "fixed".

Reply:
Whoops you'e right, missed that since I was reviewing each commit individually.